fre/make/createMakefile.py

```diff
@@ -29,8 +53,10 @@ def makefile_create(yamlfile,platform,target):
             if modelYaml.platforms.hasPlatform(platformName):
                 pass
             else:
-                raise SystemExit (platformName + " does not exist in " + modelYaml.platformsfile)
+                raise SystemExit (platformName + " does not exist in " + modelYaml.combined.get("compile").get("platformYaml"))
```

Comment:
pretty sure we dont want to exit the system just because of this error- rather, just cease execution. mirrors my comments in a few diff spots